backend/pps/sch.py

```diff
@@ -45,5 +50,25 @@ def get_cycle(self, character_stats, caster_tax):
         if ((30 - 2 * short_gcd) % (short_gcd + caster_tax) > 1.5) : 
             result += 6 * (2 * short_gcd + math.ceil((30 - 2 * short_gcd) / (short_gcd + caster_tax)) * (short_gcd + caster_tax)) - 1 * caster_tax
         else:  
-            result += 6 * (2 * short_gcd+math.floor((30-2*short_gcd) / (short_gcd+caster_tax)) * (short_gcd+caster_tax)) - 1 * caster_tax
+            result += 6 * (2 * short_gcd+math.floor((30 - 2 * short_gcd) / (short_gcd + caster_tax)) * (short_gcd + caster_tax)) - 1 * caster_tax
         return result
+
+    # MP consumed over a 3 min cycle (positive = losing mp, negative = gaining mp)
+    def get_mp(self, character_stats, caster_tax, succ, adlo, ed, rez, cycle):
+        short_gcd = character_stats.get_gcd()
+        result = 0
```

Comment:
no need to init variable in python, can probably choose a better name than "result"

Reply:
leaving this one as-is. i'm 50% sure that python style recommends initializing a variable outside a conditional if it would otherwise be initialized in conditional branches. adding linter pass soon so i'll let the linter decide whether this one stands.

result currently matches the other functions -- variable names can be changed on a cleanup pass for consistency within the file.